ttnn/ttnn/operations/core.py

```diff
@@ -309,12 +309,12 @@ def to_torch(
         tensor([[-0.3008, -0.8438,  0.3242],
                 [ 0.9023, -0.5820,  0.5312]], dtype=torch.bfloat16)
     """
+    if tensor.layout != ttnn.ROW_MAJOR_LAYOUT:
```

Comment:
we convert to row-major on-device before moving it from device because it activates untilize on device by default?

Reply:
are there differences in the output between untilize on host vs. device or is it negligible?

Reply:
if there are changes, we might want to consider adding a toggleable option for switching between host vs. device untilizer

Reply:
there is a perf hit for utilizing on host vs device. Adding this as a toggleable option is a good idea